src/Chapter2.hs

```diff
@@ -336,7 +356,14 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList start end list
+  | start < 0 = []
+  | diff  < 0 = []
+  | otherwise = take (diff + 1) (drop start list)
+  where
+    -- TODO: this is failing when "range is negative";
+    -- I'm not sure if this means the start is negative, or end, or both.
+    diff = end - start
```

Comment:
In this task we mean that if either `start` or `end` is negative, you should return an empty list as well.